SDK/CMakeLists.txt

```diff
@@ -70,5 +70,5 @@ add_subdirectory(simData)
 add_subdirectory(simVis)
 add_subdirectory(simUtil)
 if(QT_FOUND)
-    add_subdirectory(simQt)
+    #add_subdirectory(simQt)
```

Comment:
this looks like a leftover form testing

Reply:
Agree, please back out.  If you're having issues with simQt after QT_FOUND, what problem are you having?  Is it because osgQt isn't found?

Reply:
Hmm. My local copy has this reverted. Not sure how it got through.

Anyway, it's not an SDK problem -- I was not able to get osgQt built and gave up rather than waste more time on it.